value_converter.go

```diff
@@ -39,8 +39,8 @@ func (c converter) ConvertValue(v interface{}) (driver.Value, error) {
 			bytes := []byte(s)
 			return bytes, nil
 		}
-		// default behaviour to convert uint64 to int64
-		return int64(u64), nil
+		// default behaviour to convert uint64
+		return uint64(u64), nil
```

Comment:
u64 already has type uint64, so the conversion is extra
The comment also can be removed, since it does not  have value.